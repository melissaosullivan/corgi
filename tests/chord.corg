pitch p1 = 4;
pitch p2 = 5;
duration d1 = <1/4>;
duration d2 = <1/8>;
chord c = [(p1,d1), (p1,d2), (p2, d2)];