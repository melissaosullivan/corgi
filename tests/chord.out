
int main()
{
pitch p1 = 4;
pitch p2 = 5;
duration d1 = $1/4$;
duration d2 = $1/3$;
chord c = (a, b), 2, 3, Binop here, 5, 6;
}

